### ISSUE TITLE
Rtools in windows

### ISSUE THREAD

**Issue by myuen89** [STAT545-UBC/Discussion#464]
@jennybc 

Hi Jenny,

I have not yet resolved my computer problem, still using the ones at work for all my HW needs. However, hw009 requires us to install Rtools, an action I am unable to complete due to security restrictions. I would like to ask for an extension for hw009 until Friday Dec 9th, 2016, so that I may resolve this issue.

Thank you.

**Comment by jennybc** [comment]
I don't think you actually need Rtools to use devtools on Windows, if your package is pure R. It might warn you about not having it, but ... are you actually having a problem?
